src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private Date date;
+    private int workingHour;
+    private int pricePerHour;
+    private String currentName;
+    private int salary;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        String answer = "Report for period " + dateFrom + " - " + dateTo + System.lineSeparator();
+        try {
+
+            for (String name:names) {
+                for (int i = 0; i < data.length; i++) {
+                    String[] words = data[i].split(" ");
+                    date = new SimpleDateFormat("dd.MM.yyyy").parse(words[0]);
+                    currentName = words[1];
```

Comment:
![image](https://github.com/mate-academy/jv-salary-info/assets/12864128/3045ef3c-2f68-436d-8db4-a6505c5e8dcb)

move all indexes to the constant